config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
   resources :accounts, only: [:index, :create]
   resources :task_groups, only: [:index], path: "task-groups"
+  resources :tasks, only: [:create] do
+    member do
+      patch 'stop'
+      patch 'complete'
+    end
+  end
+  get 'tasks/recording', 'tasks#recording'
+  get 'tasks/pending', 'tasks#pending'
```

Comment:
```suggestion
     collection do
       get 'recording'
       get 'pending'
     end
   end
```

調べてみたところ、こんな感じで書けそうです！

Reply:
ありがとうございます！
こちらの suggestion を適用させていただきます！